### PR TITLE
[TW-726] Refreshed Dropbox integration page

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Reduced number of screenshots, and reviewed remaining screenshots for accuracy.
* Updated instructions to match previously refreshed integration pages.
* Added "Contents" section.
* Fixed Vale suggestions.

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 1]
```diff
@@ -14,54 +14,40 @@ contextual_links:
 
 Back up and synchronize your Postman Collections on Dropbox for file sharing, storage, and collaboration. The Postman to Dropbox integration seamlessly complements your existing workflows and allows you to store your Postman Collections and other project files in one place.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:15]:
```suggestion
Back up and synchronize your Postman Collections on Dropbox for file sharing, storage, and collaboration. The Postman to Dropbox integration complements your existing workflows and allows you to store your Postman Collections and other project files in one place.
```

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 2]
```diff
@@ -14,54 +14,40 @@ contextual_links:
...
-1. Select **[Integrations](https://go.postman.co/integrations)**.
+1. From the **[Home](https://go.postman.co/home)** page, select **[Integrations](https://go.postman.co/integrations)**.
 
-    ![integrations page](https://assets.postman.com/postman-docs/dropbox-integrations.jpg)
+    ![integrations page](https://assets.postman.com/postman-docs/home-integrations.jpg)
 
-1. Select **[Browse All Integrations](https://go.postman.co/integrations/browse?category=all)**.
-
-    ![dropbox search all](https://assets.postman.com/postman-docs/dropbox-search-all.jpg)
+1. Select **Browse All Integrations**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:28]:
```suggestion
1. Select **[Browse All Integrations](https://go.postman.co/integrations/browse?category=all)**.
```

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 3]
```diff
@@ -14,54 +14,40 @@ contextual_links:
...
 ## Accessing your Postman Collections on Dropbox
 
-You can find your backups on Dropbox > Apps > Postman.
+Once you've backed up your collection in Dropbox, you can find your backups at **Dropbox > Apps > Postman Pro**.
 
-  ![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.jpg)
+  ![view in dropbox](https://assets.postman.com/postman-docs/v10/dropbox-view-v10.jpg)
 
-From now on, every change saved to your Postman Collection will automatically upload changes to Dropbox in real time. Complement your team's existing workflows and back up your Postman Collections to Dropbox.
+Going forward, every change saved to your Postman Collection will automatically upload to Dropbox in real time. Complement your team's existing workflows and back up your Postman Collections to Dropbox.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:53]:
```suggestion
Going forward, every change saved to your Postman Collection will automatically upload to Dropbox.
```

File: src/pages/docs/integrations/available-integrations/dropbox.md [hunk 4]
```diff
@@ -14,54 +14,40 @@ contextual_links:
...
-1. Select **Add Integration**. Your collection is saved as a single JSON file and uploaded to Dropbox.
+1. Select **Add Integration** to save the configuration.
 
 ## Accessing your Postman Collections on Dropbox
 
-You can find your backups on Dropbox > Apps > Postman.
+Once you've backed up your collection in Dropbox, you can find your backups at **Dropbox > Apps > Postman Pro**.
 
-  ![view in dropbox](https://assets.postman.com/postman-docs/dropbox_view.jpg)
+  ![view in dropbox](https://assets.postman.com/postman-docs/v10/dropbox-view-v10.jpg)
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/dropbox.md:51]: Add a border to this screenshot

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Much better than before! 🌟

File: src/pages/docs/integrations/available-integrations/dropbox.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/dropbox.md:51]: The screenshot has a border.

**Pull request merged**